graph/mutators/comment.js

```diff
@@ -187,11 +187,33 @@ const setCommentStatus = ({loaders: {Comments}}, {id, status}) => {
     });
 };
 
+/**
+ * Adds a tag to a Comment
+ * @param {String} id          identifier of the comment  (uuid)
+ * @param {String} tag     name of the tag
+ */
+const addCommentTag = ({user, loaders: {Comments}}, {id, tag}) => {
+  return CommentsService.addTag(id, tag, user.id)
+    .then(() => CommentsService.findById( id ));
```

Comment:
I'm unsure of the best approach to this for now, possibly we can adjust the way that the query for the comment is completed, but it should definitely not live in the mutators folder. Possibly moving them to the root_mutators.js file in the resolvers folder?